pkgs/development/interpreters/php/default.nix

```diff
@@ -156,7 +158,7 @@ let
       ++ optional ( pdo_mysqlSupport || mysqliSupport ) "--with-mysql-sock=/run/mysqld/mysqld.sock"
       ++ optional bcmathSupport "--enable-bcmath"
       # FIXME: Our own gd package doesn't work, see https://bugs.php.net/bug.php?id=60108.
-      ++ optionals gdSupport [
+      ++ optionals (gdSupport && (versionOlder version "7.4")) [
```

Comment:
Did you checked that GD is now enabled?
The flag has been changed https://github.com/php/php-src/blob/PHP-7.4/UPGRADING#L739

Reply:
Thanks, fixed.